financial-templates-lib/logger/SlackTransport.js

```diff
@@ -67,14 +67,33 @@ const slackFormatter = info => {
         }
       }
     } else {
-      // If the value in the message object is an string or a integer then show it as _key: value
-      formattedResponse.blocks.push({
-        type: "section",
-        text: {
-          type: "mrkdwn",
-          text: ` • _${key}_: ${info[key]}`
-        }
-      });
+      // if the key is a transaction object, generate a transaction link
+      if (key == "tx") {
+        formattedResponse.blocks.push({
+          type: "section",
+          text: {
+            type: "mrkdwn",
+            text: ` • _tx_: <https://etherscan.io/tx/${info[key]}|${info[key]}> \n`
+          }
+        });
+      } else if (key == "markwn") {
```

Comment:
nit: Why `markwn` instead of `markdwn`?

Reply:
`markwn` is the key used in the Slack transport to define the `type`. I'm just following their convention.